02-Reproject-Raster-In-R.Rmd

```diff
@@ -280,3 +293,7 @@ plot(DSM_SJER,
 
 ```
 
+
+If you completed the San Joaquin plotting challenge in the
```

Comment:
what are you looking for here in terms of a lesson learned, etc? Maybe another hidden chunk is warranted?